styles.css

```diff
@@ -278,17 +278,16 @@ nav {
 .speaker-background {
   color: #ff4e38;
   font-style: italic;
-  font-weight: 500;
-  font-size: 10px;
+  font-weight: 700;
+  font-size: 12px;
   margin: 0;
 }
 
 .speaker-description {
   color: #252525;
-  font-weight: 300;
-  font-size: 10px;
+  font-weight: 400;
+  font-size: 13px;
```

Comment:
[OPTIONAL] Great job changing the font size in the mobile version, but it would be better to use a different size in the desktop version since you are using media queries. (For example, if you use 13px in the mobile version, in the desktop version should be 15 or 16px).